js/basicblocks.js

```diff
@@ -2013,17 +2013,17 @@ function initBasicProtoBlocks(palettes, blocks, beginnerMode) {
     }
 
     // macro
-    var newblock = new ProtoBlock('settimbre');
+    var newblock = new ProtoBlock('setinstrument');
```

Comment:
We really don't want to change the internal block name since that will break old projects. We just want to change the label itself, which you do on line 2020.

Reply:
@tanhengyeow Are you going to make Walter's suggested correction? I think you are very close.

Reply:
@pikurasa Yes, for sure. I'll work on it 😄